mmv1/products/notebooks/api.yaml

```diff
@@ -281,6 +281,37 @@ objects:
               Defines whether the instance has the vTPM enabled.
               Enabled by default.
             default_value: true
+      - !ruby/object:Api::Type::Enum
+        name: 'nicType'
```

Comment:
Given that this feature will be in public preview please add `min_version: beta` to each new field, and register a beta version at the top of this file. It should point to the REST API that this feature will be exposed under.

Reply:
ack